private-media-framework/draft-ietf-perc-private-media-framework.md

```diff
@@ -711,3 +698,362 @@ a co-author.
 
 {backmatter}
 
+# PERC Key Inventory {#keyinventory}
+
+PERC specifies the use of a number of different keys and, understandably,
+it looks complicated or confusing on the surface.  This section summarizes
+the various keys used in the system, how they are generated, and what
+purpose they serve.
+
+The keys are described in the order in which they would typically be
+acquired.
+
+The various keys used in PERC are shown in
+(@key-inventory-table) below.
+
+{#key-inventory-table align="center"}
+~~~
++-----------+----------------------------------------------------+
+| Key       | Description                                        |
++-----------+----------------------------------------------------+
+| KEK       | Key shared by all endpoints and used to encrypt    |
+| (EKT Key) | each endpoint's SRTP master key so receiving       |
+|           | endpoints can decrypt media.  Key provided by the  |
+|           | key distributor via the EKTKey message.            |
```

Comment:
I would say in here "This key is provided from the key distributor to the endpoints via ..."